modules/infinite-scroll/infinity-customizer.js

```diff
@@ -0,0 +1,54 @@
+/* globals wp */
```

Comment:
I have moved all purely Customizer logic into its own file and retained its coupling to `jQuery`, because the aim of this PR is to rewrite front-end code to vanilla JS.